.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,80 @@
+name: Docker Release
+
+on:
```

Comment:
Maybe add a dependency on the docker build first? Or consolidate docker release and build  and only run the release steps on release

Reply:
Consolidating isn't trivial because `release` and `push` have very different context available in the workflow. It is possible but would probably need a lot of ugly conditional statements.